v3.1/custom-rules/before-crs.dist/cleanlogs.conf

```diff
@@ -1,3 +1,3 @@
 # === Exempt frequent well-known requests from logging, e.g. health checks (ids: 40000 - 40100)
 
-SecRule REQUEST_FILENAME "@streq /healthz" "phase:2,id:40000,nolog,allow"
+SecRule REMOTE_ADDR "^10.1.\d+.1$" "phase:1,id:40000,nolog,allow,ctl:auditEngine=Off"
```

Comment:
Does a plain regular expression work? I would guess you need a `@rx` in front of it.

Reply:
As this is now not self-explanatory anymore I'd suggest we add a comment above the rule that explains that this is about "ignoring local traffic, such as health checks (that perform a request on `/healthz`)".

Reply:
Yep, plain RegExp does work. Copied this from a Tutorial and tested it (By adding the rule to the config map manualy)

I THOUGHT about adding a comment but apparently I’m not Yoda :-( will add an explanatory comment

Reply:
Added a comment, please review